app/utils/camera_discovery.py

```
@@ -389,33 +390,49 @@ def _local_video_devices(base_path="/dev"):
+        "local": _local_video_devices,
+    }
+
+    with ThreadPoolExecutor(max_workers=len(tasks)) as executor:
+        future_to_stage = {
+            executor.submit(func): stage for stage, func in tasks.items()
+        }
+        for fut in as_completed(future_to_stage):
+            stage = future_to_stage[fut]
+            try:
+                cameras.extend(fut.result())
+            except Exception as e:  # pragma: no cover - network
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)